tests/Aspire.Playground.Tests/Aspire.Playground.Tests.csproj

```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+
+    <IsTestProject>true</IsTestProject>
+    <!-- no docker support on helix/windows yet -->
+    <RunTestsOnHelix Condition="'$(OS)' != 'Windows_NT'">true</RunTestsOnHelix>
+    <SkipTests Condition="'$(OS)' == 'Windows_NT'">true</SkipTests>
+
+    <DeployOutsideOfRepoSupportFilesRelativeDir>staging-archive\</DeployOutsideOfRepoSupportFilesRelativeDir>
+
+    <TestArchiveTestsDir>$(TestArchiveTestsDirForBuildOnHelixTests)</TestArchiveTestsDir>
+
+    <PlaygroundSourceDir>$(MSBuildThisFileDirectory)..\..\playground\</PlaygroundSourceDir>
+    <TestsSharedDir>$(MSBuildThisFileDirectory)..\Shared\</TestsSharedDir>
+    <RunSettingsFilePath>$(MSBuildThisFileDirectory).runsettings</RunSettingsFilePath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(TestsSharedDir)WorkloadTesting\TestOutputWrapper.cs" />
+    <Compile Include="$(TestsSharedDir)WorkloadTesting\EnvironmentVariables.cs" />
+    <Compile Include="$(TestsSharedDir)Logging\*.cs" />
+
+    <AspireProjectOrPackageReference Include="Aspire.Hosting" />
+    <AspireProjectOrPackageReference Include="Aspire.Hosting.AppHost" />
+    <AspireProjectOrPackageReference Include="Aspire.Hosting.Azure" />
+    <AspireProjectOrPackageReference Include="Aspire.Hosting.NodeJs" />
```

Comment:
re:nodejs, https://github.com/dotnet/aspire/blob/575b0f8fb04bda4fb60fe6e8b16425934053dafa/tests/Aspire.Playground.Tests/Infrastructure/DistributedApplicationExtensions.cs#L145-L146

.. but we can drop the reference by checking the type name instead of the type itself.

Reply:
Azure, and AppHost can be dropped 👍

Reply:
Ok, we can leave NodeJS for now since we reference a type in it.